### PR TITLE
Prepping for 1.3.0 Release

### PR DESCRIPTION
**Pull request by keithconvictional**
- Removed event type
- Fixed spelling errors
- Update ref to use $ref (With default main) and not `$INPUT_REF` as default
- Updated the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,16 +14,103 @@ When deploying an app you may need to deploy additional services, this Github Ac
...
+    echo "For testing failure"
+    exit 1
+```
+
+
+## Potential Issues
+
+### Timing
+
+The actions dispatch is an asynchronous job and it at times can take a few seconds to start. If you do not have a delay, it may be started after you have check if it was successful. ie. Start dispatch call --> No delay --> Check if successful --> Actually starts. If the workflow has run before, it will just complete immediately as a successful run. You can solve this by simply increasing the delay to a few seconds. By default it is 10 seconds. This is also large delays between checks which helps the traffic to the Github API.
```
Comment (stephen-choi-convictional) [inline README.md:105]: nit typo: "after you have checked..." instead of "after you have check"

Comment (stephen-choi-convictional) [reply README.md:105]: nit typo: "There are also large delays.." instead of "This is also large delays"

File: entrypoint.sh [hunk 2]
```diff
@@ -7,10 +7,10 @@ function usage_docs {
   echo "    repo: myrepo"
   echo "    github_token: \${{ secrets.GITHUB_PERSONAL_ACCESS_TOKEN }}"
   echo "    workflow_file_name: main.yaml"
-}
+} # TODO - Update
```
Comment (stephen-choi-convictional) [inline entrypoint.sh:10]: Does something need to be done here?

**Comment by stephen-choi-convictional** [comment]
Might be quicker if I just ask here, does removal of event type do anything?

**Comment by keithconvictional** [comment]
Yes, technically but we changed the endpoint. The old one is seems to be deprecated and we updated it. Now, event type isn't needed.

**Review by stephen-choi-convictional — APPROVED** [review]
LGTM

**Pull request merged**